addOns/todo/src/main/java/org/zaproxy/zap/extension/todo/TodoList.java

```diff
@@ -132,8 +132,7 @@ public void actionPerformed(ActionEvent e) {
 
     public NodeList parseXml() throws Exception {
 
-        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
-        DocumentBuilder db = dbf.newDocumentBuilder();
+        DocumentBuilder db = XmlUtils.newXxeDisabledDocumentBuilderFactory().newDocumentBuilder();
```

Comment:
The file is bundled in the add-on.

Reply:
Just saying, like in the other case (does no harm and appeases the machine, so).